sympy/solvers/ode/tests/test_single.py

```diff
@@ -1825,7 +1825,10 @@ def _get_examples_ode_sol_separable():
 
     'separable_26': {
         'eq': f1 - k * (v(t) ** 2) - m * Derivative(v(t)),
-        'sol': [Eq(v(t), -68.585712797928991/tanh(C1 - 0.14288690166235204*t))],
+        'sol': [Eq(v(t), (
+            -1.0*exp(0.2857738033247041*C1 - 0.2857738033247041*t) - 1.0)
```

Comment:
This just looks bad for a CAS. Perhaps this should be handled at some low level, but if there are already floats in a factor or term, then floats that are equivalent to ints can be shown as ints, e.g.

`1.23*x + 1` not `1.23*x + 1.0`
`exp(1.23*x) + 1` not `1.0*exp(1.23*x) + 1.0` or `1.0*(exp(1.23*x) + 1)`

Perhaps this is too fine a detail to worry about.